.github/workflows/android.yml

```diff
@@ -67,27 +67,33 @@ jobs:
           - android: 11
             android-api: 30
 
-          - android: 12
-            android-api: 31
-
           - android: 12L
             android-api: 32
 
+          - android: 13
+            android-api: 33
+
```

Comment:
is it worth adding API 34 now that the API 34 support has been merged?

Reply:
I will add it later. 

Needs to be added, tested, confirmed on paramedic first before adding to plugins. I usually try and set paramedic CI first as its used as the source of truth for all plugin CI workflows.

Reply:
Went ahead and added API 34 since paramedic has been updated since. Also updated the JDK to 17.